app/src/main/res/values/strings.xml

```diff
@@ -5,27 +5,25 @@
   -->
 
 <resources xmlns:tools="http://schemas.android.com/tools">
-    <!-- This string is a promotional or short descriptional sentence to describe the purpose of the application -->
+    <!-- This string is a promotional or short descriptional sentence to describe the purpose of the application. -->
     <string name="app_tagline">Take your passwords everywhere.</string>
-    <!-- This is used for screenreaders to know the application logo image is currently selected, not expected to translate Firefox Lockbox -->
-    <string name="app_logo">Firefox Lockbox Logo</string>
+    <!-- This is used for screenreaders to know the application logo image is currently selected. %1$S will be replaced with application name. -->
+    <string name="app_logo">%1$S Logo</string>
```

Comment:
I believe all instances of the placeholder should be "%1$s" (lowercase "s") instead of "%1$S"

Reply:
>all instances of the placeholder should be "%1$s" (lowercase "s")

Whoops, right 👌